src/Common/test-runtime/LatestDependencies/project.json

```diff
@@ -0,0 +1,75 @@
+{
```

Comment:
Wait... the validation rule should _not_ try to accept * or different versions when we are doing a normal build... we really do want that validation to work, and we want package consistency. 

Maybe the correct thing to do is to put the project.json in some other directory as suggested, and condition the floating dependency project.json to include only on `$(FloatingTestRuntimeDependencies=true`, and also in dir.props, condition such that validation does _not_ happen when `$(FloatingTestRuntimeDependencies=true`

When `$(FloatingTestRuntimeDependencies=false`, then the floating dependency project.json should not factor into the build at all.